### PR TITLE
[IMP] general: email comms gmail edits

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=2923899&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

ODOO VERSION 15 only

### CODE REVIEW

File: content/applications/general/email_communication/google_oauth.rst [hunk 1]
```diff
@@ -114,8 +114,8 @@ Enter Google Credentials
 ------------------------
 
 First, open Odoo and navigate to the :guilabel:`Apps` module. Then, remove the :guilabel:`Apps`
```
Comment (ksc-odoo) [inline content/applications/general/email_communication/google_oauth.rst:116]:
```suggestion
First, open Odoo, and navigate to the :menuselection:`Apps` module. Then, remove the :guilabel:`Apps`
```

File: content/applications/general/email_communication/google_oauth.rst [hunk 2]
```diff
@@ -114,8 +114,8 @@ Enter Google Credentials
...
-filter from the search bar and type in `Google`. Install the module called :guilabel:`Google
-Gmail`.
+filter from the search bar and type in `Gmail`. Install the modules called :guilabel:`Google
```
Comment (ksc-odoo) [inline content/applications/general/email_communication/google_oauth.rst:117]:
```suggestion
filter from the :guilabel:`Search...` bar, and type in `Gmail`. Install the modules called :guilabel:`Google
```

File: content/applications/general/email_communication/google_oauth.rst [hunk 3]
```diff
@@ -114,8 +114,8 @@ Enter Google Credentials
...
+Gmail` and :guilabel:`Fetchmail Gmail`.
 
 Next, navigate to :menuselection:`Settings --> General Settings`, and under the :guilabel:`Discuss`
```
Comment (ksc-odoo) [inline content/applications/general/email_communication/google_oauth.rst:120]:
```suggestion
Next, navigate to :menuselection:`Settings app --> General Settings`, and under the :guilabel:`Discuss`
```

**Review by ksc-odoo — APPROVED** [review]
@tiku-odoo -- just finished my Final Review. I know this is super-short, so I'm approving now, because the suggestions I left are very minor. Once this is ready to go, feel free to tag ZST for Tech Review, since SALI is out-of-office at the moment. Thanks! 👍

**Comment by tiku-odoo** [comment]
@StraubCreative 

This PR is now ready for your review. It has been run through Make Review and KSC as well. 

***This change is for Odoo 15.0 only***

Thanks, 
Tim

**Pull request closed without merging**